server/build/webpack.js

```diff
@@ -171,4 +172,7 @@ export default async function createCompiler (dir, { hotReload = false } = {}) {
       return interpolateNames.get(this.resourcePath) || url
     }
   })
+  const config = await getConfig(dir)
```

Comment:
Oh right! That need was removed since we now use `require`. Good catch, I'll update the PR.